lib-injection/run-lib-injection.sh

```diff
@@ -8,11 +8,11 @@ SCRIPT_PATH=$(readlink -f "$0")
 # Absolute path this script is in, thus /home/user/bin
```

Comment:
would now be a good time to merge the `lib-injection/run.sh` and `lib-injection/run-lib-injection.sh` scripts? I think it's confusing to have both.

Reply:
renamed run.sh -> execFunction.sh